src/main/java/com/team701/buddymatcher/socket/ChatEventHandler.java

```diff
@@ -0,0 +1,60 @@
+package com.team701.buddymatcher.socket;
+
+import com.corundumstudio.socketio.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ChatEventHandler {
+
+    SocketIOServer server;
+    Map<Integer, String> userChats = new HashMap(); //Map of UserId to namespace for chat
```

Comment:
I think this might be out of scope for the issue, but each chat would have at least 2 users, so I don't think a single Integer key would work? An alternative could be to use the namespace as the key, and then the value could be the list of User IDs

Reply:
Good point, we can make that change later if needed